cluster/schema/schema.go

```diff
@@ -113,11 +113,6 @@ func (s *schema) ClassEqual(name string) string {
 }
 
 func (s *schema) unsafeClassEqual(name string) string {
-	for alias := range s.aliases {
```

Comment:
shouldn't we leave this check here? bc we are invoking this method in `ClassEqual(name string) string` to check if a given class name exists, shouldn't we also take the alias into account here?

Reply:
You are right. We should leave it. the ClassEqual is used in PreApplyFilter checks, to make sure we don't create any collection with "existing" alias name.

Removed the change.